src/blockchain/p2p/Address.hpp

```diff
@@ -96,9 +96,10 @@ class Address final : public internal::Address
         const Time lastConnected,
         const UnallocatedSet<Service>& services,
         const bool incoming) noexcept(false);
-    Address(const Address& rhs) noexcept;
 
-    ~Address() final = default;
+    ~Address() final;
+
+    Address(const Address* rhs) noexcept;
```

Comment:
One argument ctors tend to be declared `explicit` to avoid implicit conversions.

Reply:
Agreed. This whole Address business is so dirty I'll need to think a bit about its future.